src/commands/githubActions/bumpUpstream/index.ts

```diff
@@ -202,9 +202,9 @@ Compose - ${JSON.stringify(compose, null, 2)}
     });
   } catch (e) {
     e.message = `Error getting next version from apm: ${e.message}`;
-    e.message += `\nManifest version could not be updated`;
+    e.message += `\nManifest version set to default 0.1.0`;
```

Comment:
You are assuming all the errors are related due to a non published package. You should handle and catch the specifi error you are looking for and in that scenario set the manifest 0.1.0 version

Reply:
What I could do is to handle the error like:
If error is `NO_REPO` then put version 0.1.0, but if not, we should keep the version that is already set in the package and that is it.

I believe that not being able to bump the current package version should NOT make the action fail, as it is not essential that the package version is the same as the latest published (when the package is published the version will be automatically bumped)

Reply:
> If error is NO_REPO then put version 0.1.0, but if not, we should keep the version that is already set in the package and that is it.
Yes, im just saying to do that

Reply:
Fixed, if the error is `NOREPO`, I set the manifest version to `0.1.0`. In other case, the error is thrown